guten_reader_FE/Components/HomeScreen/HomeScreen.js

```diff
@@ -15,6 +15,11 @@ class HomeScreen extends React.Component {
     Font.loadAsync({
       'Roboto': require('../../assets/fonts/Roboto.ttf'),
```

Comment:
HomeScreen.js is no longer being used, so I believe we will need to move this code in the HomeScreen.js file for the Search into the Library.js component file (which is the new home after the splash page).

Reply:
My apologies, this was old code.  I don't see any particular need to hold onto this component anymore, do you?

Reply:
Nope! Thanks for deleting it!